api/v1alpha1/circuitbreaker_types.go

```diff
@@ -46,4 +46,20 @@ type CircuitBreaker struct {
 	// +kubebuilder:validation:Maximum=4294967295
 	// +optional
 	MaxRequestsPerConnection *int64 `json:"maxRequestsPerConnection,omitempty"`
+
+	// Defines per-host Circuit Breaker thresholds
+	// +optional
+	// +notImplementedHide
+	PerHostThresholds *PerHostCircuitBreakers `json:"perHostThresholds,omitempty"`
+}
+
+// PerHostCircuitBreakers defines the per-host Circuit Breaker configuration.
+type PerHostCircuitBreakers struct {
```

Comment:
Nit: 
It looks to me the field name `PerHostThresholds` and the type name`PerHostCircuitBreakers` are misaligned.

How about?
1. `PerHost` and  `PerHostThresholds`
2. `PerHost` and  `PerHostCircuitBreakers`

Reply:
I prefer option 2 and updated with that